sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -35,3 +36,47 @@ case class Size(child: Expression) extends UnaryExpression with ExpectsInputType
     nullSafeCodeGen(ctx, ev, c => s"${ev.primitive} = ($c).size();")
   }
 }
+
+case class ArrayContains(left: Expression, right: Expression) extends BinaryExpression with ExpectsInputTypes {
+  override def dataType: DataType = BooleanType
+
+  override def inputTypes: Seq[AbstractDataType] = Seq(ArrayType, AnyDataType)
```

Comment:
The error I am getting from python, using the unit test in `functions.py`, is that `bigint` is not of type `int`. So I was checking the type in the prior code version, but `acceptsType` was not smart enough to see that those are comparable. So either need to implement that, or see where there is a utility to do that without using the implicit conversions trait since that supports a much larger class of conversions